src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.html

```diff
@@ -24,7 +24,8 @@ <h2>Iris</h2>
                     </div>
                     <div *ngIf="isServerSentMessage(message)" style="width: fit-content">
                         <div class="client-chat">
-                            <span [innerHTML]="content.textContent! | htmlForMarkdown"></span>
+                            <span *ngIf="isFirstMessage && i == 0" [innerHTML]="content.textContent! | artemisTranslate"></span>
+                            <span *ngIf="!isFirstMessage || (isFirstMessage && i > 0)" [innerHTML]="content.textContent! | htmlForMarkdown"></span>
```

Comment:
Why are these 2 case distinctions necessary? Wouldn't the joint condition to show innerHTML be  (isFirstMessage && i == 0) || (!isFirstMessage || (isFirstMessage && i > 0)) = !isFirstMessage || (isFirstMessage && i >= 0)

Reply:
Because otherwise the 'artemisTranslate' will try to translate all messages